airflow/api/common/experimental/__init__.py

```diff
@@ -41,7 +41,7 @@ def check_and_get_dag(dag_id: str, task_id: str | None = None) -> DagModel:
     return dag
 
 
-def check_and_get_dagrun(dag: DagModel, execution_date: datetime) -> DagRun:
+def check_and_get_dagrun(dag: DagModel, execution_date: datetime):
```

Comment:
Don’t remove this

Reply:
Done. For context, I removed it when I was working on removing `get_dagrun` and `get_task_instances`. But since I reverted these changes, I can add it back.